src/Components/Results/ProgramPage/ProgramPage.tsx

```diff
@@ -110,6 +111,22 @@ const ProgramPage = ({ program }: ProgramPageProps) => {
   };
   const value = useFormatYearlyValue(program);
 
+  const warningMessages = useMemo(() => {
+    return program.warning_messages.filter((warningMessage) => {
+      if (warningMessage.legal_statuses.length === 0) {
+        return true;
+      }
```

Comment:
What do you think about adding a comment here to denote that this if statement handles a `warning_message` that is not tied to a legal status and therefore applies generally.